contracts/RevenueDistributionToken.sol

```diff
@@ -162,8 +163,9 @@ contract RevenueDistributionToken is IRevenueDistributionToken, ERC20 {
     }
 
     function _burn(uint256 shares_, uint256 assets_, address receiver_, address owner_, address caller_) internal {
-        require(shares_ != 0, "RDT:B:ZERO_SHARES");
-        require(assets_ != 0, "RDT:B:ZERO_ASSETS");
+        require(receiver_ != address(0), "RDT:M:ZERO_RECEIVER");
```

Comment:
Would it make sense to also check if the `owner_` is set to `address(0)` here?

Reply:
`RDT:B:ZERO_RECEIVER`